### DISCUSSION TITLE
What's the purpose of setting tag `latest`?

### DISCUSSION THREAD

**Discussion by amazingfate** [containerd/accelerated-container-image#278] [General]
I'm trying  to package for debian, when running `dh-make-golang -type dev github.com/containerd/accelerated-container-image` I got the following error:
```
2024/04/15 19:25:44 Starting "dh-make-golang v0.6.0 linux/amd64"
2024/04/15 19:25:44 Downloading "github.com/containerd/accelerated-container-image/..."
2024/04/15 19:26:04 Determining upstream version number
2024/04/15 19:26:04 Found latest tag "latest"
2024/04/15 19:26:04 WARNING: Latest tag "latest" is not a valid SemVer version
2024/04/15 19:26:04 Latest tag "latest" matches master
2024/04/15 19:26:04 Package version is ""
2024/04/15 19:26:04 Determining dependencies
2024/04/15 19:26:07 Could not determine repo path for import path "oras.land/oras-go/v2/content/oci": unrecognized import path "oras.land/oras-go/v2/content/oci"
2024/04/15 19:26:08 Downloading https://github.com/containerd/accelerated-container-image/archive/latest.tar.gz
2024/04/15 19:26:13 Moving tempfile to "golang-github-containerd-accelerated-container-image_.orig.tar.gz"
```
No version is detected.

**Comment by BigVan** [comment]
Actually I think the 'latest' version is unnecessary also.
cc @yuchen0cc

**Comment by BigVan** [comment]
@amazingfate I think you may need to specify a stable version to compile?

**Reply by amazingfate** [reply]
I tried to package for debian, and `dh-make-golang` will try to find  the latest tag by default. There is an option `-git_revision` to set git revision.
Happy to see tag `latest` has been deleted,  thank you very much!

**Comment by yuchen0cc** [comment]
The `latest` tag is related to the `latest pr`, and it is a `Pre-release`. The `latest` tag does not point to any `release` tag.
Please try to use release tag instead.
